diagnostic.md

```diff
@@ -94,4 +94,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+_exercise, Jason, JasonWeeks_
```

Comment:
1 correct out of 3 potential. Exercise is correct.
```javascript
'teamMember['name'] = {
  given: 'Jason',
  surname: 'Weeks'
};
``` 
It takes the whole object of name and shows the keys and values. 

```javascript
teamMember['name']['surname'] = 'Weeks';
```

It finds the object of 'name' and points to the key in that object of 'surname' and returns it's value of 'Weeks'